### ISSUE TITLE
Svn installation not found.

### ISSUE THREAD

**Issue by lcyan** [JohnstonCode/svn-scm#389]
- VSCode Version: 1.28.2(user setup)
- OS Version: window7 6.1.7601 Service Pack 1 Build 7601
- Extension Version: 1.44.1
- System Language:zh-cn
- SVN Version: 1.10.3 (r1842928)

## Issue ##
Svn installation not found.

but 

![_20181018125405](https://user-images.githubusercontent.com/3824082/47132163-1bbc6a80-d2d5-11e8-87c1-f1e87b079385.png)
 
my vscode setting 

"svn.path": "C:\\Program Files\\TortoiseSVN\\bin\\svn.exe",

## Steps to Reproduce ##

1. open vscode
2. show svn not found tip

## SVN Output ##
```
Paste here content of "View >> Output >> OUTPUT >> SVN"
Svn installation not found.
Updated "svn.path" with "c:\Program Files\TortoiseSVN\bin\svn.exe"
Svn installation not found.
```

## Screenshots ##
![_20181018125615](https://user-images.githubusercontent.com/3824082/47132217-6d64f500-d2d5-11e8-900d-a24b8936786e.png)
![_20181018125628](https://user-images.githubusercontent.com/3824082/47132218-6d64f500-d2d5-11e8-9440-eb430ed52a14.png)

**Comment by JohnstonCode** [comment]
Unable to replicate on my windows machine. Was this working on 1.44.0?

**Comment by JohnstonCode** [comment]
You also shouldn't need to specify the path in settings as `C:\Program Files\TortoiseSVN\bin\svn.exe` is one of the paths it looks in

**Comment by lcyan** [comment]
> Unable to replicate on my windows machine. Was this working on 1.44.0?

working on 1.44.1

**Comment by JohnstonCode** [comment]
Are you still having a problem?

**Comment by lcyan** [comment]
> Are you still having a problem?

Yes.

![_20181019094641](https://user-images.githubusercontent.com/3824082/47193166-179b5600-d384-11e8-91aa-551afffb6d71.png)

**Comment by edgardmessias** [comment]
@lcyan, run follow codes and paste here the response:

```cmd
c:\Program Files\TortoiseSVN\bin\svn.exe --version
echo %errorLevel%
c:\Program Files\TortoiseSVN\bin\svn.exe help checkout
echo %errorLevel%
```

Join on [Gitter](https://gitter.im/svn-scm/Lobby)

**Comment by edgardmessias** [comment]
The problem is length of help command

**Comment by lcyan** [comment]
> The problem is length of help command

thanks

**Comment by alisalxy** [comment]
hi,
 I also miss the issure "SVN not found"
![svn](https://user-images.githubusercontent.com/44763455/50948350-26a22800-14dd-11e9-81f7-c3e988071bfd.PNG)


Below is the relevant information about vscode：
![vs code](https://user-images.githubusercontent.com/44763455/50948692-57369180-14de-11e9-8313-a75a994fe334.PNG)

I follow the instructions above to execute the results as follows:


C:\Program Files\TortoiseSVN\bin>svn.exe --version
[0x7FEFD15EFF7] ANOMALY: use of REX.w is meaningless (default operand size is 64)
svn, version 1.11.0 (r1845130)
   compiled Oct 30 2018, 21:50:05 on x86-microsoft-windows

Copyright (C) 2018 The Apache Software Foundation.
This software consists of contributions made by many people;
see the NOTICE file for more information.
Subversion is open source software, see http://subversion.apache.org/

The following repository access (RA) modules are available:

* ra_svn : Module for accessing a repository using the svn network protocol.
  - with Cyrus SASL authentication
  - handles 'svn' scheme
* ra_local : Module for accessing a repository on local disk.
  - handles 'file' scheme
* ra_serf : Module for accessing a repository via WebDAV protocol using serf.
  - using serf 1.3.9 (compiled with 1.3.9)
  - handles 'http' scheme
  - handles 'https' scheme

The following authentication credential caches are available:

* Wincrypt cache in C:\Users\Administrator\AppData\Roaming\Subversion


C:\Program Files\TortoiseSVN\bin>echo %errorLevel%
0


C:\Program Files\TortoiseSVN\bin>svn.exe help checkout
[0x7FEFD15EFF7] ANOMALY: use of REX.w is meaningless (default operand size is 64)
checkout (co): Check out a working copy from a repository.
usage: checkout URL[@REV]... [PATH]

  If specified, REV determines in which revision the URL is first
  looked up.

  If PATH is omitted, the basename of the URL will be used as
  the destination. If multiple URLs are given each will be checked
  out into a sub-directory of PATH, with the name of the sub-directory
  being the basename of the URL.

  If --force is used, unversioned obstructing paths in the working
  copy destination do not automatically cause the check out to fail.
  If the obstructing path is the same type (file or directory) as the
  corresponding path in the repository it becomes versioned but its
  contents are left 'as-is' in the working copy.  This means that an
  obstructing directory's unversioned children may also obstruct and
  become versioned.  For files, any content differences between the
  obstruction and the repository are treated like a local modification
  to the working copy.  All properties from the repository are applied
  to the obstructing path.

  See also 'svn help update' for a list of possible characters
  reporting the action taken.

Valid options:
  -r [--revision] ARG      : ARG (some commands also take ARG1:ARG2 range)
                             A revision argument can be one of:
                                NUMBER       revision number
                                '{' DATE '}' revision at start of the date
                                'HEAD'       latest in repository
                                'BASE'       base rev of item's working copy
                                'COMMITTED'  last commit at or before BASE
                                'PREV'       revision just before COMMITTED
  -q [--quiet]             : print nothing, or only summary information
  -N [--non-recursive]     : obsolete; try --depth=files or --depth=immediates
  --depth ARG              : limit operation by depth ARG ('empty', 'files',
                             'immediates', or 'infinity')
  --force                  : force operation to run
  --ignore-externals       : ignore externals definitions

Global options:
  --username ARG           : specify a username ARG
  --password ARG           : specify a password ARG (caution: on many operating
                             systems, other users will be able to see this)
  --password-from-stdin    : read password from stdin
  --no-auth-cache          : do not cache authentication tokens
  --non-interactive        : do no interactive prompting (default is to prompt
                             only if standard input is a terminal device)
  --force-interactive      : do interactive prompting even if standard input
                             is not a terminal device
  --trust-server-cert      : deprecated; same as
                             --trust-server-cert-failures=unknown-ca
  --trust-server-cert-failures ARG : with --non-interactive, accept SSL server
                             certificates with failures; ARG is comma-separated
                             list of 'unknown-ca' (Unknown Authority),
                             'cn-mismatch' (Hostname mismatch), 'expired'
                             (Expired certificate), 'not-yet-valid' (Not yet
                             valid certificate) and 'other' (all other not
                             separately classified certificate errors).
  --config-dir ARG         : read user configuration files from directory ARG
  --config-option ARG      : set user configuration option in the format:
                                 FILE:SECTION:OPTION=[VALUE]
                             For example:
                                 servers:global:http-library=serf
								 
								 
C:\Program Files\TortoiseSVN\bin>echo %errorLevel%
0

thank you so much!

**Comment by hannahbellesheart** [comment]
I know this has been closed a while but i figured you would probably want this on the same thread as opposed to opening a new one...

I have put the path into my system path, trid several different iterations of environment variables. i have updated the svn.path in vscode settings.and have tried multiple variations, like path via environment variables, actual paths, short-code paths like `C:\PROGRA~1` .....Ive had my devbox (a Dell Precision Laptop Workstation) for about 6 or 7 years and I installed Tortoise SVN on the devbox the day I got it. It has never stopped running and has never failed. In fact I have been using TortoiseSVN for 17 years, on every dev machine Ive ever had. And on this particular devbox, even after upgrading from Windows 10 to Windows 11... My Tortoise SVN still runs just fine. I installed your plugin about a month ago and I have gotten this error message ever since I installed it. And again, my SVN still runs fine. The extension however, does not.

![image](https://github.com/user-attachments/assets/75b829f1-e54b-4f12-b6a1-0ac66a03eda5)

![image](https://github.com/user-attachments/assets/3c5f3510-e55c-43d7-9977-2cd14637e6ab)
